patches/roslyn/0007-Don-t-use-Humanizer-in-source-build.patch

```diff
@@ -0,0 +1,92 @@
+From f44e01f25b4d45a6a66a739f8ba7b89d7c52d63e Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Thu, 24 Sep 2020 13:45:50 -0500
+Subject: [PATCH 7/7] Don't use Humanizer in source-build.
```

Comment:
Do you know what the package does? Are there any inhuman results on the outputs? 😄

Reply:
It appears to be used to help make friendly names of things, for instance expressions in the debugger.  Some of these will end up being grammatically incorrect (plurals) or harder to read (CamelCaseWithNoSpaces), but there should be no real functional impact.

Reply:
Interesting... it sounds like this might be important for new users in particular though--reduce the number of weird things they need to understand to investigate issues they're seeing. I think we should track this as a product experience bug/diff. Looks like it's in .NET Foundation at least, to add it at some point: https://github.com/Humanizr/Humanizer

Reply:
Logged https://github.com/dotnet/source-build/issues/1767 to track it.